accounts/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from .models import *
+
+# Register your models here.
+
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
+    list_display = ('email', 'is_staff', 'is_active',)
+    add_fieldsets = (
+        (None, {
+            'classes': ('wide',),
+            'fields': ('email', 'username','password1', 'password2', 'is_staff', 'is_active')}
+        ),
+    )
```

Comment:
**For purposes of documentation a comment would be necessary to highlight this implementation.**